doc/source/merge_changelogs.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Append release notes from other projects of the larray-project constellation to the current changelog
+"""
+
+import os
+import requests
```

Comment:
move this below, this is 3rd party lib, not a a python stdlib (even if it should ;-))